packages/documentation-v7/package.json

```diff
@@ -42,6 +42,7 @@
     "@storybook/addon-essentials": "7.1.1",
     "@storybook/addon-links": "7.1.1",
     "@storybook/blocks": "7.1.1",
+    "@storybook/client-api": "6.5.16",
```

Comment:
v6.5 sounds suspicious ?! Isn't there a v7 version?

Reply:
You're right, it's superseded by @storybook/preview-api. That package is not documented as of now and I did not use either of them anyway (wanted to sync checked status with the control but it's a little complicated so I left it as is for the moment. It would be nice to have but not critical IMHO).